### PR TITLE
updated franz (0.9.10)

### PR DESCRIPTION
**Pull request by karl82**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/franz.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'franz' do
-  version :latest
-  sha256 :no_check
+  version '0.9.10'
+  sha256 'bac5896e5cad84d4018e9486cd580a04548c0a0b350ade506b82192175306570'
 
   # bit.ly/22yh2b6 was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/franz.rb]: Comment needs fixing.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in c13e9e7d63a57c6cc5f8c0399581f91e539bcfd0. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by karl82** [comment]
Ah, I was just fixing it... Never mind. Next time

**Comment by vitorgalvao** [comment]
No worries. I had my terminal open and I had just finished fixing another cask, so I thought “why not”.

**Pull request closed without merging**